### DISCUSSION TITLE
[résolu] 16. Sécuriser l'accès à l'administration

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#9] [Bonnes pratiques] [answered]
Je ne sais pas comment tourner le principe de validation sur cette fiche.
[16. Sécuriser l'accès à l'administration](https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/16.%20S%C3%A9curiser%20l'acc%C3%A8s%20%C3%A0%20l'administration.md)

**Comment by hrenaud** [comment]
**Proposition**

| Le nombre de pages donnant l'accès à l'interface d'administration | est inférieur ou égal à |
| ------------- | :---------------------: |
| avec l'url `votresite.net/admin`        |            0            |
| avec l'url `votresite.net/login`        |            0            |

**Reply by dNicolle** [reply]
Bonne idée, je vais aller dans ce sens. J'avais omis qu'on pouvait mettre plusieurs lignes :)

**Reply by dNicolle** [reply]
@florinesueur cette nomenclature ne semble pas correspondre avec ton point là dessus.
dit moi si c'est bon ? ou s'il faut mettre le "pages donnant l'accès à l'interface..." sur chaque ligne ?

**Comment by florinesueur** [comment]
| Le nombre  | est inférieur ou égal à |
| ------------- | :---------------------: |
| de pages donnant l'accès à l'interface d'administration avec l'url `votresite.net/admin`        |            0            |
| de pages donnant l'accès à l'interface d'administration avec l'url `votresite.net/login`        |            0            |

ça donnerai ça. C'est ainsi fait dans le référentiel.

**Comment by dNicolle** [comment]
ok, je change :)

Le ven. 30 sept. 2022 à 18:16, Florine Sueur ***@***.***> a
écrit :

**Reply by florinesueur** [reply]
Après ne perdez pas de temps la dessus. Je le ferais au besoin lors de la relecture. C'était pour être sûre qu'on procède ainsi ou non.

**Comment by florinesueur** [comment, marked as answer]
Une issue a été créée la mise à jour de cette BP
